### PR TITLE
Add default parameters in the waitForPodsReady configuration

### PR DESCRIPTION
**Pull request by tenzen-y**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
I added the default parameters to the waitForPodsReady configuration.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: charts/kueue/values.yaml [hunk 1]
```diff
@@ -79,7 +79,13 @@ managerConfig:
       burst: 100
     #pprofBindAddress: :8082
     #waitForPodsReady:
-    #  enable: true
+    #  enable: false
+    #  timeout: 5m0s
```
Comment (gabesaba) [inline charts/kueue/values.yaml:83]: can this just be 5m, without 0s?

File: charts/kueue/values.yaml [hunk 2]
```diff
@@ -79,7 +79,13 @@ managerConfig:
...
+    #  blockAdmission: false
+    #  requeuingStrategy:
+    #    timestamp: Eviction
+    #    backoffLimitCount: null // null indicates infinity requeuing
```
Comment (gabesaba) [inline charts/kueue/values.yaml:87]: nit: s/infinity/infinite

**Comment by tenzen-y** [comment]
Thank you for the review!
@gabesaba @mimowo  All comments make sense. I addressed all comments, PTAL.

**Comment by gabesaba** [comment]
lgtm

**Comment by mimowo** [comment]
/lgtm
Thanks

**Pull request merged**